picard/tagger.py

```diff
@@ -1246,12 +1246,58 @@ def sighandler(self):
         self.signalnotifier.setEnabled(True)
 
 
-def version():
-    print("%s %s %s" % (PICARD_ORG_NAME, PICARD_APP_NAME, PICARD_FANCY_VERSION_STR))
+class PicardArgs:
 
+    def __init__(self, argparse_args=None):
```

Comment:
Is there a reason to make `argparse_args` optional?

Reply:
Well, actually no.